packages/duckdb/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "uvu test test.js"
   },
   "dependencies": {
-    "duckdb-async": "0.6.1",
```

Comment:
Was your package-lock.json file also updated?
If so can you commit that too?

Reply:
Can you point me to the package-lock.json file you're referring to? I don't see one in an obvious place in the main branch of evidence that I forked. (I'm an NPM noob, so may be missing something obvious here 😁 )

Reply:
(I suspect this is what is causing the tests to fail)
` ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with packages/duckdb/package.json`

Reply:
gotcha. Just committed that.